docs/get_started.rst

```diff
@@ -1,17 +1,353 @@
-Get started
+Get Started
 ===========
 
+.. contents:: 
+    :depth: 2
+    :local:
+
+
 Install CumulusCI
 -----------------
 
+.. note:: These installation instructions assume some familiarity with entering commands into a terminal.
+    If that's completely new to you, we recommend starting with the `CumulusCI setup module <https://trailhead.salesforce.com/content/learn/modules/cumulusci-setup>`_ module on Trailhead to walk you through step by step.
+
+
+On macOS 
+^^^^^^^^
+Ensure you have `Homebrew <https://brew.sh/>`_ installed as it is a prerequisite for installing CumulusCI on macOS.
+
+
+Install via ``pipx`` 
+******************************
+Follow `installation instructions for pipx <https://pipxproject.github.io/pipx/installation/>`_.
```

Comment:
It's just 2 commands (brew install pipx and pipx ensurepath) -- but may require opening a new terminal before the pipx command is actually available. For Mac I'm inclined to include the commands inline rather than linking to the pipx docs. Less context switching, and less chance of them trying to install using pip instead of homebrew (which means we don't know what python they're using).